synapse/http/replicationagent.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2023 The Matrix.org Foundation C.I.C.
```

Comment:
Is there a place to grab a current license header for new files?

Reply:
(I just copy & paste every time.)